### PR TITLE
Draft: feat: add @ima/testing-library

### PR DESCRIPTION
**Pull request by Filipoliko**

### CODE REVIEW

File: packages/cli/src/webpack/utils.ts [hunk 1]
```diff
@@ -226,7 +226,7 @@ export function requireImaConfig(rootDir = process.cwd()): ImaConfig | null {
  * @param {ImaCliArgs} args CLI args.
  * @returns {Promise<ImaConfig>} Ima config or empty object.
  */
-export async function resolveImaConfig(args: ImaCliArgs): Promise<ImaConfig> {
+export function resolveImaConfig(args: ImaCliArgs): ImaConfig {
```
Comment (Filipoliko) [inline packages/cli/src/webpack/utils.ts:229]: If we cannot / should not make this function async, then we have to rethink `@ima/testing-library` method `render` to be either also async (and we will have to call `await render` in tests), or we have to separate context (or more specifically language) initialisation process into individual method and by this, remove the option to call `render` without any additional setup.

**Comment by Filipoliko** [comment]
@mjancarik Is `@ima/devtools` still being used? There are some enzyme tests, which need to be rewritten to RTL, but there is no IMA specific logic, so there is no need for `@ima/testing-library`.

File: packages/cli/src/webpack/utils.ts (hunk 1 above)
Comment (Filipoliko) [reply packages/cli/src/webpack/utils.ts]: It is better to be prepared for async future, these changes have been reverted and `@ima/testing-library` is more async friendly now.

File: packages/create-ima-app/template/common/app/component/card/__tests__/CardSpec.js [hunk 2]
```diff
@@ -10,8 +10,8 @@ describe('Card', () => {
       children: 'Some content of the card.',
     };
 
-    const wrapper = shallow(<Card {...props} />);
+    const { container } = render(<Card {...props} />);
 
-    expect(wrapper).toMatchSnapshot();
+    expect(container).toMatchSnapshot();
```
Comment (mjancarik) [inline packages/create-ima-app/template/common/app/component/card/__tests__/CardSpec.js:15]: container.firstChild?

Comment (Filipoliko) [reply packages/create-ima-app/template/common/app/component/card/__tests__/CardSpec.js]: changed to `container.firstChild`

**Review by mjancarik — APPROVED** [review]

**Pull request merged**